pdsfile/pdsfile_test_helper.py

```diff
@@ -6,10 +6,11 @@
 ##########################################################################################
 
 from .pdsfile import abspath_for_logical_path
-import pprint
 import ast
+from collections import OrderedDict
```

Comment:
We don't need to use `OrderedDict`. Python 3 guarantees that the order of dictionary keys will stay the same. This was only needed for Python 2.